lib/src/utils/resolveRichTextToNodes.spec.ts

```diff
@@ -225,6 +232,12 @@ describe("resolveNode", () => {
       },
       content: [{ content: "Hello from rich text" }],
     });
+
+    it("empty heading", () => {
+      expect(resolveNode(emptyNode)).toStrictEqual({
+        component: "br",
+      });
+    });
```

Comment:
it seems we have here `describe in describe` situation. it is not quite necessarily bad, but inconsistent at the moment.

The whole file is done in this structure:
```
describe('resolveNode', () => {
  it('heading', () => {
    expect(...).toStrictEqual({ ... });
    expect(...).toStrictEqual({ ... });
    expect(...).toStrictEqual({ ... });
  });
  it('paragraph', () => { ... });
  it('hard_break', () => { ... });
  ...
});

describe('resolveMark', () => {
  it('link', () => { ... });
  it('bold', () => { ... });
  ...
});
```

so following that you might simply go with:

```
// empty content
expect(resolveNode(emptyNode)).toStrictEqual({
   component: "br",
 });
```
or

```
// empty content
expect(resolveNode({
      type: "heading",
      attrs: {
        level: 2,
      },
    })).toStrictEqual({
   component: "br",
 });
```
to be extra concise.

Reply:
Yup, totally! It could be an improvement in terms of failed testing readability, but that's definitely out of the scope of this PR.